sig-ml-ops/charter.md

```diff
@@ -0,0 +1,53 @@
+# Charter
+
+This charter adheres to the conventions described in the [Open Data Hub Charter Guide](../sig-charter-guide.md) and uses the Roles and Organization Management outlined in [sig-goverance](../sig-governance.md).
+
+## Scope
+
+To create services for deploying and managing machine learning models and pipelines in OpenShift and across the hybrid cloud. Our users are defined as data scientists who are interested in incorporating ML Ops capabilities into their workflows.
+
+### In scope
+
+* Services and APIs for pipelines and model serving ecosystem
```

Comment:
I think we should expand on this. There are two high level use cases at play:
1) Data scientist using pipelines to iterate over experimentation on a model
2) Data scientist deploying a model to a serving environment, potentially using pipelines

I think we should make it clear that both of these are in scope.